zeppelin-web/src/app/spell/spell-base.js

```diff
@@ -31,9 +31,10 @@ export class SpellBase {
    * Consumes text and return `SpellResult`.
    *
    * @param paragraphText {string} which doesn't include magic
+   * @param config {Object}
    * @return {SpellResult}
    */
-  interpret(paragraphText) {
+  interpret(paragraphText, config) {
```

Comment:
Another idea came up. 

- Setting config before calling `interpret` like `spell.setConfig(conf)`
- Then user can access config like this 

```javascript
interpret(paragraphText) {
  const conf = this.config; // or getConfig()? 
}
```